src/mysql.php

```diff
@@ -278,7 +285,7 @@ function mysql_result($result, $row, $field = 0)
                         $found = true;
                         break;
                     }
-                    $i++;
+                    ++$i;
```

Comment:
This is unnecessary; PHP will perform this optimization transparently (if there is nothing on the LHS), and it introduces cognitive overhead, at least in my mind — `$i++` is just easier to grok, right to left.

This applies to all occurrences of this change.